documentation/modules/proc-creating-configmap.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// assembly-external-logging.adoc
+
+[id='creating-configmap_{context}']
+= Creating a ConfigMap for logging
+
+To use a ConfigMap to define logging properties, you create the ConfigMap and then reference it as part of the logging definition in the `spec` of a resource.
+
+The ConfigMap must contain the appropriate logging configuration.
+
+* `log4j.properties` for Kafka components
+* `log4j2.properties` for the Topic Operator and User Operator
```

Comment:
should we say that the above `log4j.properties` and `log4j2.properties` are the fields in the ConfigMap data section where the file content should be put?

Reply:
Nice idea! Thanks